### PR TITLE
Add all metadata values to config endpoint

### PR DESCRIPTION
**Pull request by joel-wright**

### CODE REVIEW

File: backend/app/api/api_v1/routers/lookups/main.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 )
 from app.api.api_v1.schemas.metadata import Config
 from app.db.session import get_db
+from backend.app.db.models.document import Framework, Hazard, Keyword, Response
```
Comment (diversemix) [inline backend/app/api/api_v1/routers/lookups/main.py:14]:
```suggestion
from app.db.models.document import Framework, Hazard, Keyword, Response
```

Comment (diversemix) [reply backend/app/api/api_v1/routers/lookups/main.py:14]: This is the reason its failing

Comment (joel-wright) [reply backend/app/api/api_v1/routers/lookups/main.py:14]: damn IDE does this from time to time

**Review by diversemix — APPROVED** [review]
lgtm

File: backend/app/api/api_v1/routers/lookups/main.py (hunk 1 above)
Comment (joel-wright) [reply backend/app/api/api_v1/routers/lookups/main.py]: sorted

**Pull request merged**